members.md

```diff
@@ -22,4 +22,4 @@ List of 2019/2020 MIPT-V project members:
 21. Vitaly Nizamutdinov
 22. Aleksandr Troegubov
 23. Grigoriy Egorov
-24. Yaroslav Okatev
+24. Yaroslav Okatev
```

Comment:
Please revert that change since it is not related to PR.

Reply:
I'm trying to do this, but after 'git commit' I get new lines in members.md:
<<<<<<< HEAD
=======
<<<<<<< HEAD
=======
>>>>>>> 82fedd60ea4740a91f8667083af970ea04c9acff
=======

Reply:
> Please revert that change since it is not related to PR.

Very sad, but I still can’t revert commit No `698eee5` ...